src/main/java/by/antonsh/lesson3/Ex1.java

```diff
@@ -0,0 +1,11 @@
+package by.antonsh.lesson3;
+
+public class Ex1 {
+    public static void main(String[] args) {
+        int km=10;
+        for (int i = 1; i < 7; i++) {
+            km *=1.1;
```

Comment:
it is 6 days. does result is int when we multiply by 1.1?

Reply:
fix it